### ISSUE TITLE
Live now and next up marquees don't respect reduce screen motion settings

### ISSUE THREAD

**Issue by antiantivirus** [Refuge-Worldwide/refuge-worldwide#72]
When user turns on reduce screen motion settings these marquees should be paused in order to respect the users settings.
